lib/main_micro_alp.pm

```diff
@@ -119,6 +114,7 @@ sub load_installation_tests {
         }
         loadtest 'console/textinfo';
     }
+    replace_opensuse_repos_tests if is_repo_replacement_required;
```

Comment:
Probably needs to be inside the `else` above, otherwise this will break [microos-10G](https://openqa.opensuse.org/tests/3236496)

Reply:
valid point

Reply:
(IMO the HDD 10G schedule should be separate anyway)

Reply:
http://kepler.suse.cz/tests/20604#

Reply:
> (IMO the HDD 10G schedule should be separate anyway)

Yeah, I think we can fix that in another PR.